frontend/src/metabase/css/dashboard.css

```diff
@@ -263,24 +263,6 @@
   font-size: inherit;
 }
 
-.AutomaticDashboard.AutomaticDashboard--withSidebar #SmartScalar-PreviousValue {
-  display: none !important;
-}
-
-@media screen and (min-width: 1280px) {
-  .AutomaticDashboard.AutomaticDashboard--withSidebar
-    #SmartScalar-PreviousValue {
-    display: none;
-  }
-}
-
-@media screen and (min-width: 1540px) {
-  .AutomaticDashboard.AutomaticDashboard--withSidebar
-    #SmartScalar-PreviousValue {
-    display: inherit;
-  }
-}
-
```

Comment:
I removed this because we have a new system for responsively toggling this value when it doesn’t fit in the card.

_Extra context— the id `#SmartScalar-PreviousValue` is only used here as a hack to hide long numbers at the end of `↑ 50% · vs. previous month: ______` on automated dashboards (via x-ray) when the page isn’t wide enough, but this is a hack because it doesn’t measure the card width, which our new system does instead._